### PR TITLE
Update teams-live-event-usage-report.md

### PR DESCRIPTION
**Pull request by bwilsonms**
Adding that anonymous presenters will not be included in the report.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-analytics-and-reports/teams-live-event-usage-report.md [hunk 1]
```diff
@@ -42,7 +42,9 @@ The Teams live event usage report in the Microsoft Teams admin center shows you
 |**5**   |Select **Edit columns** to add or remove columns in the table.|
 
 ## Notes
-We show up to 100 live events that match the current report criteria. To see more live events, apply date filters to reduce the list size. 
+We show up to 100 live events that match the current report criteria. To see more live events, apply date filters to reduce the list size.
+
+Anonymous presenters will not be included in report.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-live-event-usage-report.md:47]:
```suggestion
Anonymous presenters will not be included in the report.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bwilsonms, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@HowlinWolf-92 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**